wiki_to_csv.py

```diff
@@ -88,6 +102,12 @@ def extract_all_data_on_page(page, year):
                             .format(page, key)
                         )
                         continue
+                    elif not is_int(value):
```

Comment:
Can this be replaced by [`value.isnumeric()`](https://docs.python.org/3/library/stdtypes.html#str.isnumeric) (or isdigit() or isdecimal(), not sure what the actual difference is)?

Reply:
I guess that for this usage case the main difference is that this only accepts integers whereas `isdecimal` (why is it not snake_case!) Which is closest also allows decimal numbers.

This is a copy-paste from some other code where it also makes use of the feature that value can be None, int or str. But here, by design, it should always be str.

Reply:
@lokal-profil to clarify: we never expect decimal numbers so they are also a sign something has gone wrong (e.g. a thousand separator might have been added)

Reply:
It seems that for this case, any of the is...() methods mentioned above will work. "Decimal" is apparently not something-with-a-decimal-point, see https://stackoverflow.com/questions/44891070/whats-the-difference-between-str-isdigit-isnumeric-and-isdecimal-in-python. There's some maths going on here that's beyond me, but those methods should work so 🤷.

Reply:
@sebastian-berlin-wmse since we only allow base 10 positive integers we cn be more restrictive than either if them are. E.g. 3³ would be true for some of those but nonsense in our use case. Similarly both 3.14 and 3,14 make no sense as input for us.